api/src/main/java/org/apache/iceberg/transforms/Bucket.java

```diff
@@ -43,6 +43,12 @@ static <T> Bucket<T> get(int numBuckets) {
     return new Bucket<>(numBuckets);
   }
 
+  /**
+   * Instantiates a new Bucket Transform
+   *
+   * @deprecated use {@link #get(int)} instead; will be removed in 2.0.0
```

Comment:
minor: the deprecated msg is typically the other way around: `will be removed in 2.0.0; use {@link #get(int)} instead`

Reply:
Thanks, I think this how you typically structure a sentence in Dutch 😁